bin/build-sprite-string.js

```diff
@@ -0,0 +1,32 @@
+import defaultAttrs from '../src/default-attrs.json';
+
+const svgStartTag = `<svg xmlns="${defaultAttrs.xmlns}">
+  <defs>\n`;
+const svgEndTag = ' </defs>\n</svg>';
+
+/**
+ * Renders the inner sprites as SVG Symbols
+ * @param {object} icons the icons object
+ * @returns {string} the rendered string with SVG symbols
+ */
+function buildSpriteString(icons) {
+  const symbols = Object.keys(icons)
+    .map(icon => toSvgSymbol(icon, icons[icon]))
+    .reduce((spriteString, symbolString) => spriteString + symbolString, '');
+
+  return svgStartTag + symbols + svgEndTag;
+}
+
+/**
+ * Renders a SVG symbol tag
+ * @param {string} name The name of the icon
+ * @param {string} contents The contents of the icon
+ * @returns {string} the rendered SVG symbol
+ */
+function toSvgSymbol(name, contents) {
+  return `    <symbol id="${name}" viewBox="${defaultAttrs.viewBox}">
```

Comment:
Is there a reason we need to have add indentation to the strings? It might help with performance if we omit unneeded whitespace.

Reply:
It was just for reading the svg. But since this is a build artifact, you are correct. I'm out now but I'm gonna fix this too.